roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -210,6 +210,7 @@ kiali_defaults:
   kiali_feature_flags:
     istio_injection_action: true
     istio_upgrade_action: false
+    skip_subset_validation: false
```

Comment:
move this down to the `validations` section - under line 235.

I also think you can rename it since it will be under "validations" so it will be assumed it is related to validations. So you can call this "skip_subsets" or something.